develop-guide/writeblitz/writeblitz.rst

```diff
@@ -246,6 +246,60 @@ other existing apis.
         function: get_traffic_stream_objects
         ...
 
+rest
+____
+
+The ``rest`` action allows you to make rest call to any endpoint on a device. Rest uses http protocols to 
```

Comment:
what are you referring to as `endpoint`?

Reply:
uri rest endpoint si 1.1.1.1/api/mo/sys/bgp.json. From after ip that is called endpoint

Reply:
And for with cli, still take advantage what does it mean? I deleted that section, it seemed unnecessary to write that part